src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReleaseDataFileServiceTest.cs

```diff
@@ -845,72 +845,63 @@ await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContex
+                Name = "Test data",
+                Release = release,
+                File = new File
+                {
+                    RootPath = Guid.NewGuid(),
+                    Filename = "test-data.csv",
+                    Type = FileType.Data,
+                    SubjectId = subject.Id,
+                    Created = DateTime.UtcNow,
+                    CreatedById = _user.Id
+                }
             };
-            var metaFile = new File
+            var metaReleaseFile = new ReleaseFile
             {
-                RootPath = Guid.NewGuid(),
-                Filename = "test-data.meta.csv",
-                Type = Metadata,
-                SubjectId = subject.Id
+                Release = release,
+                File = new File
+                {
+                    RootPath = Guid.NewGuid(),
+                    Filename = "test-data.meta.csv",
+                    Type = Metadata,
+                    SubjectId = subject.Id
+                }
             };
 
             var contentDbContextId = Guid.NewGuid().ToString();
-            var statisticsDbContextId = Guid.NewGuid().ToString();
-
             await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContextId))
             {
-                await contentDbContext.AddRangeAsync(
-                    new ReleaseFile
-                    {
-                        Release = release,
-                        File = dataFile
-                    },
-                    new ReleaseFile
-                    {
-                        Release = release,
-                        File = metaFile
-                    }
-                );
+                await contentDbContext.AddRangeAsync(dataReleaseFile, metaReleaseFile);
                 await contentDbContext.SaveChangesAsync();
             }
 
-            await using (var statisticsDbContext = InMemoryStatisticsDbContext(statisticsDbContextId))
-            {
-                await statisticsDbContext.AddAsync(subject);
-                await statisticsDbContext.SaveChangesAsync();
-            }
-
             var blobStorageService = new Mock<IBlobStorageService>(MockBehavior.Strict);
             var dataImportService = new Mock<IDataImportService>(MockBehavior.Strict);
 
             await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContextId))
             await using (var statisticsDbContext = InMemoryStatisticsDbContext(statisticsDbContextId))
             {
                 blobStorageService
-                    .Setup(s => s.CheckBlobExists(PrivateReleaseFiles, dataFile.Path()))
+                    .Setup(s => s.CheckBlobExists(PrivateReleaseFiles, dataReleaseFile.File.Path()))
                     .ReturnsAsync(true);
 
                 blobStorageService
-                    .Setup(s => s.GetBlob(PrivateReleaseFiles, dataFile.Path()))
+                    .Setup(s => s.GetBlob(PrivateReleaseFiles, dataReleaseFile.File.Path()))
```

Comment:
You can use `dataReleaseFile.Path()` here and all places below throughout this test class